components/collector/src/source_collectors/junit/tests.py

```diff
@@ -40,7 +40,10 @@ def _include_entity(self, entity: Entity) -> bool:
     def __entity(case_node: Element, case_result: str) -> Entity:
         """Transform a test case into a test case entity."""
         class_name = case_node.get("classname", "")
+        host_name = case_node.get("hostname", "")
         name = case_node.get("name", "unknown")
-        key = f"{class_name}:{name}"
-        old_key = name  # Key was changed after Quality-time 5.16.1, enable migration of user entity data
-        return Entity(key=key, old_key=old_key, name=name, class_name=class_name, test_result=case_result)
+        key = f"{class_name}:{host_name}:{name}"
+        old_key = f"{class_name}:{name}"  # Key was changed after v5.25.0, enable migration of user entity data
```

Comment:
Does this mean that `v5.26.0` is also a version containing migrations?

Reply:
Good question. We didn't mark the previous Junit key change as such in v5.16.2, but I guess that's an oversight. I'll update the versioning policy.